src/events/Cargo.toml

```diff
@@ -5,6 +5,6 @@ authors = ["Valerian G. <valerian.garleanu@pm.me>"]
 edition = "2018"
 
 [dependencies]
-pushevent = "0.1.3"
+pushevent = { git = "https://github.com/vgarleanu/pushevent" }
```

Comment:
Same here

Reply:
im publishing pushevent to crates.io once im done with the refactor and tls integration there.